pipeline/signed-compose-listener.groovy

```diff
@@ -115,19 +116,24 @@ node("rhel-8-medium || ceph-qe-ci") {
 
             sharedLib.SendUMBMessage(msgContent, overrideTopic, "ProductBuildDone")
             println "Updated UMB Message Successfully"
+            currentBuild.result = "SUCCESS"
         }
     } catch(Exception err) {
-        if (currentBuild.result == "ABORTED") {
-            println "Aborting the workflow."
+        if (currentBuild.result != "ABORTED") {
+            currentBuild.result = "FAILURE"
+            failureReason = err.getMessage()
+            println "Failure Reason: ${failureReason}"
         }
-
-        // notify about failure
-        currentBuild.result = "FAILURE"
-        def failureReason = err.getMessage()
-        def subject =  "[CEPHCI-PIPELINE-ALERT] [JOB-FAILURE] - ${env.JOB_NAME}/${env.BUILD_NUMBER}"
-        def body = "<body><h3><u>Job Failure</u></h3></p>"
-        body += "<dl><dt>Jenkins Build:</dt><dd>${env.BUILD_URL}</dd>"
-        body += "<dt>Failure Reason:</dt><dd>${failureReason}</dd></dl></body>"
+    } finally {
```

Comment:
Is there a way to fetch RC build information and put it in tthe email?

Reply:
Just add some more information about RC and it would be good if we mention RC in Subject line.

Reply:
Made Changes as per comment